iidm-network-api/src/main/java/eu/itesla_project/iidm/network/EquipmentTopologyVisitor.java

```diff
@@ -12,7 +12,7 @@
  */
 public abstract class EquipmentTopologyVisitor extends AbstractTopologyVisitor {
 
-    public abstract void visitEquipment(Connectable eq);
+    public abstract <I extends Connectable<I>> void visitEquipment(Connectable<I> eq);
```

Comment:
Would it be better to use wild card type (Connectable<?> here as we want to iterate on every kind of connectable without being able to define precisely the subclass?